en/streaming-integrator/docs/setup/installing-si-as-a-linux-service.md

```diff
@@ -0,0 +1,119 @@
+# Installing Streaming Integrator as a Linux Service
+
+WSO2 Streaming Integrator can be run as a Linux service.
+
+## Before you begin
+
+Install JDK version 1.8.0_144 and set the `JAVA_HOME` variable.
+
+## Download and install the Streaming Integrator
+
+Go to the WSO2 Enterprise Integrator [product page](https://wso2.com/integration/#), click **Download**, and then click **Zip Archive** to download the product distribution as a ZIP file.
```

Comment:
Isn't this should point to SI download page? (https://wso2.com/integration/streaming-integrator/ )

Reply:
Changed this as suggested.